java-snapshot-testing-spock/src/main/groovy/au/com/origin/snapshots/SpockConfig.groovy

```diff
@@ -0,0 +1,23 @@
+package au.com.origin.snapshots
+
+import org.apache.commons.lang3.NotImplementedException
+
+import java.lang.reflect.Method
+
+class SpockConfig implements SnapshotConfig {
+
+    @Override
+    String getTestSrcDir() {
+        return "src/test/groovy/"
```

Comment:
Should this be configurable? is the default / how to change documented?

Reply:
Yeah, I realised I regressed this this morning by introducing the annotations which abstract the SnapshotConfig object away . I'm actually thinking it would be nice it we didn't need it at all as it is just used for writing the file.  I'm pretty sure we can derive this in some other way.

Reply:
Perhaps we can allow the client to override the test annotation and supply their own custom config.  I this this will be most flexible.  Still think above property should go away however.

Reply:
@dpweberza This is fixed now: https://github.com/origin-energy/java-snapshot-testing/pull/7/commits/882a7bcb61cfd1dc2408a63bc5498c15ab6f832e

Reply:
Fixed